sale_automatic_workflow/models/sale_order.py

```diff
@@ -32,7 +32,7 @@ def _compute_all_qty_delivered(self):
                 or float_compare(
                     line.qty_delivered, line.product_uom_qty, precision_digits=precision
                 )
-                == 0
+                >= 0
```

Comment:
FTR this is ok on https://github.com/OCA/sale-workflow/blob/14.0/sale_delivery_state/models/sale_order.py